features/step_definitions/cluster_configuration/localstorage.rb

```diff
@@ -1,245 +1,94 @@
-# OCP supports fileystem local storage provisioner from 3.7
-# For some OCP versions, before depolying local storage provisoner,
-# we need enable some feature gates first.
-# ref: doc/storage_automation_spec.adoc
-
-Given /^I deploy local storage provisioner(?: with "([^ ]+?)" version)?$/ do |img_version|
+Given /^local storage operator has been installed successfully$/ do
```

Comment:
Why not use previous step name, just remove version stuff? This step name doesn't speak well whether installation is performed now or we just check that there is such installation. Old name made clear that we do install now.

Reply:
Same with the other steps. We better make clear in step name whether we install now or we just check it is installed.

Reply:
This step is split into 3 steps, one is local storage operator deployment, the second one is local storage provisioner deployment, the third one is creating some PVs.

For each step, it will check if the desired result is fulfilled first, if yes, it will do nothing, else it will do something to reach the desired result.

Reply:
ok